server/test/uk/gov/ons/addressIndex/server/controllers/AddressControllerSpec.scala

```diff
@@ -155,6 +156,8 @@ class AddressControllerSpec extends PlaySpec with Results {
     source = "7666OW"
   )
 
+  // todo add test highlighting
```

Comment:
When?

Reply:
I was intending to leave it empty for now (whilst we're in iterative mode), and as the highlighting only applies to partials it's likely to cause quite a bit of trouble / failed tests.